extensions/panache/panache-hibernate-common/deployment/pom.xml

```diff
@@ -17,6 +17,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-core-deployment</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-envers</artifactId>
+        </dependency>
```

Comment:
Could we add it as optional ?
I don't know if it's a bug library or not, but if it adds a lot of dependencies we should make it optional

Reply:
the build actually fails due to a missing module without it.  Not that things don't compile.  but literally maven stops and complains.  it's the first time I've ever seen that kind of error.  but I'll try marking it as provided and see what happens.

Reply:
`optional` seems better than `provided`, but I wonder why you need it ...

Reply:
actually that looks to be vestigial. i removed it and it built locally so I think some other pom changes i made rectified the "need" for that one.

Reply:
Cool!